tests/safeds/data/tabular/transformation/test_discretizer.py

```diff
@@ -36,7 +56,32 @@ def test_should_not_change_original_transformer(self) -> None:
 
 
 class TestTransform:
-    def test_should_raise_if_column_not_found(self) -> None:
+    @pytest.mark.parametrize(
+        ("table_to_transform", "error", "error_message"),
+        [
+            (
+                Table(
+                    {
+                        "col2": ["a", "b", "c"],
+                    },
+                ),
+                UnknownColumnNameError,
+                r"Could not find column\(s\) 'col1'",
+            ),
```

Comment:
Please add a test that tests if the correct error message is raised when there are multiple missing columns. See #407 for how you can sort the column names.